cg/meta/workflow/fluffy.py

```diff
@@ -196,13 +196,6 @@ def run_fluffy(self, case_id: str, dry_run: bool) -> None:
     def get_bundle_created_date(self, case_id: str) -> dt.datetime:
         return self.get_date_from_file_path(self.get_samplesheet_path(case_id=case_id))
 
-    @staticmethod
```

Comment:
Did this slip in? Since it's in the fluffy workflow and all.

Reply:
This mades sense as a general AnalysisAPI method (previously only fluffy used it, now its re-used also by mutant)